src/main/kotlin/me/lensvol/blackconnect/BlackdClient.kt

```diff
@@ -14,12 +14,15 @@ sealed class BlackdResponse {
     data class UnknownStatus(val code: Int, val responseText: String) : BlackdResponse()
 }
 
-class BlackdClient(val hostname: String, val port: Int) {
+class BlackdClient(val hostname: String, val port: Int, val https: Boolean) {
```

Comment:
https seems like an odd name to be honest
I would either expect protocol (a string that default has a value 'http') or a boolean like useSsl

Reply:
I would agree, please consider naming this as something like `useSSL` or `connectOverHttps`. :)

Reply:
I think keeping it as 'https' in the GUI seems most logical and user friendly.
@lensvol is the location ok for you? => how can i improve the space in between the boxes?
![image](https://user-images.githubusercontent.com/22102283/104030808-3e940b80-51cc-11eb-9bd9-ea63e3864889.png)